src/Diagram.js

```diff
@@ -280,9 +252,16 @@ class Diagram extends React.PureComponent {
     const { width, height } = this.containerRef.current
       ? this.containerRef.current.getBoundingClientRect()
       : DEFAULT_CONTAINER_RECT;
+    const viewport = getViewport(scroll.left, scroll.top, width, height, zoom);
+    const viewportWithOverscanning = {
+      xMin: viewport.xMin - (this.props.overscanPixels ? this.props.overscanPixels.width : width),
+      xMax: viewport.xMax + (this.props.overscanPixels ? this.props.overscanPixels.width : width),
+      yMin: viewport.yMin - (this.props.overscanPixels ? this.props.overscanPixels.height : height),
+      yMax: viewport.yMax + (this.props.overscanPixels ? this.props.overscanPixels.height : height),
```

Comment:
add `@babel/plugin-proposal-nullish-coalescing-operator` in `.babelrc` `plugins`

Reply:
done

Reply:
@satyamgaursprinklr need to add this in package.json as a devDependency too